modules/packages/MPI.chpl

```diff
@@ -49,18 +54,22 @@ Note that on Cray systems, this is not necessary, since the default
 compilers include the necessary MPI information.
 
 .. note ::
-  #. Currently, we only support the gnu compilers wrapped by MPI. We expect
-     to broaden this class in the future.
+  #. Currently, only the gnu compilers wrapped by MPI are supported. This class
+     is expected to broaden in the future.
   #. The current scripts fail to correctly determine the architecture for
-     the mpi-gnu suite, hence ``CHPL_TARGET_ARCH=none``. We expect this to be
+     the mpi-gnu suite, hence ``CHPL_TARGET_ARCH=none``. This is expected to be
```

Comment:
I didn't see this sentence in the render. Did you mean to remove it from the PR? Also, if it is removed, then you can pull the first sentence out of a list.